bootstrap/051_materialization_status.sql

```diff
@@ -0,0 +1,102 @@
+
+create or replace function internal.success_to_status(success boolean) returns text
+as
+$$
+    IFF(success IS NULL, NULL, IFF(success, 'SUCCESS', 'FAILURE'))
+$$;
+
+
+create or replace view admin.materialization_status
```

Comment:
what is the perf of this? It feels like it would take a while to compile. I assume it will take a long time to execute (with all the joins) if hte table is large

Reply:
~900ms for the 20k rows we have in the Sandbox account.

Reply:
Then I reran it later and it was more like 1.8s, roughly split equally between compilation, queuing, and execution.